backend-lambda/index.js

```diff
@@ -9,38 +9,42 @@ const headers = {
 };
 
 export const handler = async (event, context) => {
+  const body =
+    typeof event.body === "string" ? JSON.parse(event.body) : event.body;
   console.info("event", event);
   console.info("context", context);
   try {
     if (!pool) {
-      const connectionString = process.env.DATABASE_URL;
+      const connectionString = process.env.DATABASEURL;
+      console.log(connectionString);
       pool = new pg.Pool({
         connectionString,
-        application_name: "$ docs_lambda_node",
-        max: 1,
+        // application_name: "$ docs_lambda_node",
+        max: 3,
       });
     }
   } catch (e) {
     console.error(JSON.stringify(e));
   }
   let response;
-
+  // get all products
```

Comment:
This probably needs to go between 22/23

Reply:
Each comment corresponds to the event.path below it correct?